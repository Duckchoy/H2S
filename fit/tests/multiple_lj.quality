-21117.50047 -19469.06162 0.2386866021
-20651.31719 -19472.91209 0.1824016432
-20338.6646 -19473.40941 0.1821665035
-19970.07909 -19473.66228 0.1540107974
-19859.50223 -19473.23056 0.1770901115
-19650.57184 -19471.3645 0.2461646212
-19952.46515 -19473.74408 0.1834546702
-19845.92866 -19473.50973 0.181134859
-19766.99469 -19473.06261 0.1576947666
-19662.1645 -19472.30654 0.1572564139
-19627.00187 -19471.42344 0.1274873928
-19552.59605 -19472.26098 0.1614681087
-19524.79926 -19471.36149 0.1711038626
-19496.17236 -19471.80548 0.1436593298
-20217.92732 -19487.34243 0.2215584649
-19514.12624 -19486.79659 0.1401297033
-19469.14284 -19469.06162 0.2386866021
-19472.65074 -19472.91209 0.1824016432
-19473.55083 -19473.40941 0.1821665035
-19473.38471 -19473.66228 0.1540107974
-19473.12176 -19473.23056 0.1770901115
-19472.41581 -19471.3645 0.2461646212
-19473.75901 -19473.74408 0.1834546702
-19473.25813 -19473.50973 0.181134859
-19472.86678 -19473.06261 0.1576947666
-19472.3253 -19472.30654 0.1572564139
-19472.13941 -19471.42344 0.1274873928
-19471.74202 -19472.26098 0.1614681087
-19471.59323 -19471.36149 0.1711038626
-19471.44083 -19471.80548 0.1436593298
-19487.82442 -19487.34243 0.2215584649
-19486.60234 -19486.79659 0.1401297033
-19469.27802 -19469.06162 0.2386866021
-19472.61663 -19472.91209 0.1824016432
-19473.4906 -19473.40941 0.1821665035
-19473.36637 -19473.66228 0.1540107974
-19473.12819 -19473.23056 0.1770901115
-19472.47987 -19471.3645 0.2461646212
-19473.7769 -19473.74408 0.1834546702
-19473.25913 -19473.50973 0.181134859
-19472.86103 -19473.06261 0.1576947666
-19472.31768 -19472.30654 0.1572564139
-19472.13286 -19471.42344 0.1274873928
-19471.74045 -19472.26098 0.1614681087
-19471.59449 -19471.36149 0.1711038626
-19471.44561 -19471.80548 0.1436593298
-19487.73296 -19487.34243 0.2215584649
-19486.64183 -19486.79659 0.1401297033
