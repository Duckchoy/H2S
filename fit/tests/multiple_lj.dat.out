prob -119.5827622

 new best -19473.19086 -19471.34051 -19487.06369 390.1934541 14.55725998 258.9618404 -3.619402388 -119.5827622
prob -204.8331138
prob -120.3067605
prob -169.9255275
prob -91.81214053

 new best -19472.89023 -19471.34461 -19486.95273 -110.9922462 12.66040342 695.8803069 -1.967634933 -91.81214053
prob -126.7538019
prob -186.4849536
prob -115.8493225
prob -188.9153712
prob -125.7604245
prob -201.920803
prob -158.8081283
prob -155.532045
prob -167.9800184
prob -108.0512165
prob -126.2339005
prob -95.4189607
prob -181.3962521
prob -187.166981
prob -218.0553373
prob -104.9520606
prob -50.58895472

 new best -19472.16169 -19471.34398 -19486.684 -571.002798 8.613720358 1689.759199 1.868591459 -50.58895472
prob -121.8657676
prob -102.2046868
prob -209.8542495
prob -175.0667076
prob -144.9166149
prob -180.7346601
prob -112.5992497
prob -132.7715239
prob -164.1807564
prob -129.6786659
prob -61.62645883
prob -132.5537934
prob -172.0240753
prob -176.8717192
prob -172.9706776
prob -134.9312935
prob -198.1804822
prob -186.1173302
prob -155.9980126
prob -181.3779146
prob -79.93900328
prob -155.5194862
prob -142.0544346
prob -118.008742
prob -189.6396547
prob -124.1380921
prob -181.3671826
prob -103.5764712
prob -151.4056943
prob -90.88894245
prob -119.1065482
prob -124.1745097
prob -133.2081926
prob -178.7677983
prob -194.1307603
prob -166.2321744
prob -121.3444061
prob -206.8813305
prob -140.4493818
prob -132.3234648
prob -74.6361012
prob -97.95028469
prob -100.1389957
prob -162.8155062
prob -183.05428
prob -93.85841568
prob -88.58843475
prob -83.5365203
prob -185.7119769
prob -164.3513422
prob -152.9880523
prob -142.768895
prob -149.1289574
prob -125.6505656
prob -254.2416049
prob -145.3746414
prob -198.8841943
prob -94.404464
prob -153.675109
prob -132.9179112
prob -127.4960713
prob -101.8421178
prob -133.9617678
prob -193.4096532
prob -209.9997094
prob -86.36028643
prob -151.5107882
prob -155.2050672
prob -133.0878027
prob -99.86932377
prob -175.6458952
prob -135.7956727
prob -189.7414359
prob -210.9296055
prob -126.3315579
prob -68.00547125
prob -121.5798901
prob -127.3839213
prob -109.3195895
prob -120.0564261
prob -171.6048382
prob -207.4771526
prob -137.2518283
prob -168.7197064
prob -144.7513203
prob -119.1300847
prob -193.9602184
prob -154.0794709
prob -99.75596203
prob -228.8319713
prob -120.8531255
prob -156.7679699
prob -186.1637982
prob -166.9110598
prob -110.3962336
prob -166.9838351
prob -162.9025216
prob -168.5204091
prob -212.6973646
prob -114.7038894
prob -121.9620955
prob -156.4312621
prob -163.4139548
prob -199.4893102
prob -233.3892877
prob -136.7505377
prob -162.2513266
prob -114.2166581
prob -137.5826365
prob -185.9642422
prob -108.7176654
prob -127.1052118
prob -74.79233986
prob -160.8010999
prob -135.1829577
prob -141.9583101
prob -160.2269775
prob -146.2104188
prob -110.9296265
prob -178.9729624
prob -156.7196213
prob -130.7755396
prob -128.2940148
prob -104.4131787
prob -243.5865951
prob -105.7048967
prob -115.2476167
prob -148.8622633
prob -100.7050726
prob -160.5424396
prob -176.8220367
prob -85.16135192
prob -104.6019291
prob -98.47873832
prob -99.14503254
prob -77.70636745
prob -289.6824834
prob -198.3331565
prob -141.5202267
prob -245.8123443
prob -124.881477
prob -74.42628435
prob -123.3573978
prob -173.0859466
prob -166.2842225
prob -165.6416555
prob -152.8280303
prob -165.6751942
prob -73.89601741
prob -189.4252857
prob -160.1917257
prob -95.47505422
prob -156.5588797
prob -189.5459312
prob -76.68474594
prob -107.767112
prob -235.4201226
prob -182.4022541
prob -88.49039624
prob -137.5788323
prob -161.7032389
prob -121.3697589
prob -136.0486524
prob -165.8070658
prob -149.5805399
prob -126.2438339
prob -159.8396598
prob -178.6557963
prob -205.1182867
prob -121.7770568
prob -85.34388916
prob -179.8017417
prob -130.3408231
prob -188.3997663
prob -136.5725102
prob -191.563748
prob -192.533347
prob -179.3484369
prob -137.7628125
prob -265.2367711
prob -173.9252091
prob -169.0763005
prob -173.5503927
prob -217.0901069
prob -85.46257761
prob -147.1937798
prob -113.4795783
prob -127.5594671
prob -103.9870352
prob -151.4520469
prob -108.1527516
prob -108.0468935
prob -159.2346623
prob -340.0073849
prob -178.3917277
prob -137.9122532
prob -194.7854906
prob -122.6430041
prob -137.5498341
prob -178.1332953
prob -141.6618038
prob -132.3977241
prob -106.3850758
prob -78.41882637
prob -71.94899137
prob -170.0717139
prob -128.1753753
prob -107.7850989
prob -113.4754226
prob -147.3940656
prob -117.9989386
prob -99.12542306
prob -213.926455
prob -127.9356296
prob -126.3955538
prob -245.6020631
prob -180.6633667
prob -228.1406747
prob -142.295032
prob -113.452473
prob -211.7035361
prob -219.0866681
prob -106.2965442
prob -210.6926215
prob -148.4354774
prob -105.3598058
prob -103.9387634
prob -88.64297897
prob -235.7648585
prob -106.912518
prob -167.9373405
prob -148.26267
prob -198.6597792
prob -160.6779136
prob -158.4142474
prob -97.87913197
prob -138.2172706
prob -141.6063373
prob -118.8028124
prob -135.4681385
prob -129.5148153
prob -135.8414798
prob -94.16390289
prob -173.4461438
prob -146.9392837
prob -136.1492549
prob -126.8377598
prob -96.47111266
prob -171.9779289
prob -98.02681904
prob -95.0693058
prob -237.1487859
prob -88.75731979
prob -103.3222028
prob -136.1908244
prob -139.8102514
prob -117.3754331
prob -161.2086906
prob -145.3429563
prob -112.7820441
prob -178.4954239
prob -97.24305273
prob -166.4845379
prob -76.00192019
prob -141.5375668
prob -116.4020696
prob -135.9005009
prob -177.0822716
prob -128.9562467
prob -148.7585619
prob -170.5567111
prob -128.9901656
prob -143.2234027
prob -155.9054065
prob -100.5003737
prob -171.0085548
prob -149.4552442
prob -162.9391026
prob -157.3492626
prob -106.9619508
prob -121.0383744
prob -251.3085353
prob -123.0212783
prob -115.5290034
prob -84.53525255
prob -196.9675995
prob -146.3224769
prob -250.09616
prob -124.3581852
prob -125.9937589
prob -100.321474
prob -110.1438605
prob -159.0314287
prob -155.3439712
prob -138.8899918
prob -163.4081618
prob -216.2471111
prob -153.0304481
prob -103.6384115
prob -201.0189954
prob -170.8488677
prob -152.9167625
prob -128.6570052
prob -221.1050567
prob -229.6264003
prob -169.7626269
prob -175.9493347
prob -168.5668413
prob -218.5919903
prob -127.969523
prob -128.0067221
prob -207.2713645
prob -168.3175399
prob -285.8862617
prob -81.14744393
prob -106.01672
prob -138.9370579
prob -87.65765967
prob -123.3901658
prob -111.1879991
prob -111.8697747
prob -183.250592
prob -130.1506832
prob -102.7770577
prob -164.9692935
prob -176.5393533
prob -182.4548683
prob -147.6096269
prob -137.345284
prob -127.8685122
prob -165.3791041
prob -161.0330939
prob -106.7210059
prob -121.7998622
prob -236.613776
prob -143.2417003
prob -197.9722654
prob -133.3112982
prob -178.4282288
prob -136.3386532
prob -226.7876527
prob -124.1852921
prob -151.2971644
prob -147.307994
prob -126.6653876
prob -65.65469242
prob -156.0804273
prob -254.4176815
prob -192.1472224
prob -177.3060055
prob -128.3370342
prob -92.26755712
prob -158.3422161
prob -74.03461641
prob -183.1289696
prob -191.6029555
prob -79.23850407
prob -93.84346889
prob -150.5709467
prob -139.7727869
prob -173.9868275
prob -107.1111754
prob -224.2022668
prob -109.1467713
prob -126.4386483
prob -93.31806145
prob -135.6513256
prob -111.8074064
prob -105.4324129
prob -138.2379701
prob -221.1737689
prob -108.1664061
prob -175.4048578
prob -132.8446332
prob -162.1553724
prob -185.1809359
prob -116.0992781
prob -217.6663652
prob -179.7111646
prob -124.593724
prob -182.9678522
prob -146.7080536
prob -146.8638642
prob -80.8666651
prob -139.9555824
prob -164.5596863
prob -135.0643833
prob -86.72364478
prob -206.0950007
prob -136.2466499
prob -164.2308489
prob -124.5387204
prob -78.9731614
prob -124.9663187
prob -261.0819173
prob -84.95437606
prob -136.6867357
prob -74.10016499
prob -129.4923447
prob -69.72536285
prob -199.6715351
prob -122.8745504
prob -187.976881
prob -114.5988815
prob -165.8295601
prob -98.7604529
prob -157.6726137
prob -125.2444907
prob -115.7698829
prob -145.4015393
prob -162.9567898
prob -81.59180227
prob -173.3789009
prob -127.7299618
prob -84.30974795
prob -208.3743688
prob -117.5133913
prob -123.6625187
prob -129.3534458
prob -171.7664839
prob -219.5972421
prob -160.4461385
prob -156.1019374
prob -166.7114679
prob -104.9804686
prob -213.3405634
prob -142.2038033
prob -113.1990313
prob -127.2312918
prob -190.555808
prob -208.7145949
prob -132.6647463
prob -184.2460926
prob -178.4860743
prob -167.0558894
prob -121.2768419
prob -149.5355735
prob -261.93592
prob -108.0165496
prob -197.8815852
prob -112.6209422
prob -157.6625398
prob -123.2390319
prob -119.0985447
prob -180.0142154
prob -91.71821432
prob -243.7795153
prob -128.6464966
prob -184.0404153
prob -122.0965289
prob -207.9328958
prob -152.523484
prob -159.8862122
prob -143.7624649
prob -231.9600999
prob -152.4147709
prob -184.0485647
prob -189.6960496
prob -93.39322223
prob -188.8161028
prob -149.4195029
prob -153.6783415
prob -87.06538441
prob -174.1481448
prob -180.6863781
prob -75.03880982
prob -142.0063716
prob -79.82714148
prob -254.6324719
prob -121.9927346
prob -89.952306
prob -185.476094
prob -110.4402661
prob -125.4346244
prob -246.0909933
prob -164.7634
prob -126.813224
prob -145.2135101
prob -75.70669265
prob -88.29834234
prob -139.1764693
prob -182.6291709
prob -211.3636181
prob -105.6694698
prob -142.5215777
prob -119.6424471
prob -144.5266555
prob -116.9973738
prob -150.9478482
prob -183.6367661
prob -140.4865117
prob -140.7193302
prob -157.0893098
prob -93.22997984
prob -133.5881512
prob -176.6710616
prob -131.1349685
prob -139.0427712
prob -127.1493452
prob -75.65272487
prob -224.1484855
prob -169.5756833
prob -105.7917649
prob -181.9480872
prob -151.8217656
prob -135.7352632
prob -106.6874586
prob -204.2002651
prob -168.1031771
prob -142.467314
prob -161.7208734
prob -131.6654466
prob -93.05592541
prob -113.1206796
prob -148.9979879
prob -223.7586792
prob -134.3756078
prob -103.8351716
prob -133.1167678
prob -142.3966211
prob -128.0809876
prob -176.4813264
prob -173.0398871
prob -101.5424071
prob -165.8249659
prob -189.6804055
prob -234.0733272
prob -150.8839712
prob -150.4909743
prob -205.7831609
prob -148.6577607
prob -134.4391511
prob -163.0808679
prob -102.0654443
prob -142.2959056
prob -109.4716283
prob -102.8849006
prob -117.0760446
prob -207.2564814
prob -155.0691177
prob -81.6100158
prob -123.2106898
prob -126.1373337
prob -122.694827
prob -96.42790608
prob -126.9794918
prob -122.8001627
prob -133.8132242
prob -106.9412915
prob -140.7904735
prob -156.5609589
prob -177.8023063
prob -145.683048
prob -153.9335434
prob -95.01164349
prob -102.4680592
prob -212.521929
prob -191.8031727
prob -140.5924081
prob -103.6044807
prob -138.962106
prob -181.9585413
prob -117.7672865
prob -136.7714386
prob -224.0496077
prob -107.1652416
prob -108.1874437
prob -77.31490851
prob -94.53327393
prob -152.0715777
prob -162.4493288
prob -300.2485462
prob -187.3576205
prob -96.01523322
prob -201.988392
prob -162.2884036
prob -114.426885
prob -299.5962591
prob -165.1858268
prob -117.7109172
prob -110.5687015
prob -158.6531862
prob -130.4088265
prob -166.7023736
prob -98.01079452
prob -111.164579
prob -134.9669635
prob -198.3672258
prob -149.8182904
prob -203.9714688
prob -180.8317122
prob -134.7197388
prob -149.2925852
prob -132.512206
prob -188.1339789
prob -121.9473862
prob -174.151402
prob -141.3745211
prob -157.7218723
prob -129.7455335
prob -131.2632779
prob -97.37444527
prob -70.26790753
prob -201.1878808
prob -117.5668889
prob -104.7948953
prob -171.6539818
prob -117.7955474
prob -156.1607138
prob -235.5730124
prob -136.230426
prob -161.2515417
prob -240.5693056
prob -109.9421852
prob -130.7496134
prob -88.57653038
prob -179.6020915
prob -218.381643
prob -78.3656842
prob -136.0687864
prob -120.7273653
prob -166.8041679
prob -101.4874974
prob -87.09372487
prob -144.7327659
prob -165.4999365
prob -181.5045355
prob -87.15141071
prob -125.4187237
prob -133.154891
prob -131.7012968
prob -96.29438721
prob -109.6703785
prob -175.8320649
prob -110.8391589
prob -96.8527741
prob -156.9668147
prob -85.47036707
prob -167.9445719
prob -217.3290118
prob -146.4239305
prob -161.4963665
prob -111.4266613
prob -182.9938014
prob -98.72385065
prob -190.1845171
prob -66.80338681
prob -110.5741755
prob -108.9126409
prob -102.0523584
prob -180.7599242
prob -167.8227282
prob -211.7683094
prob -170.3985795
prob -81.62340992
prob -199.1336519
prob -217.5109929
prob -93.52715508
prob -136.3334962
prob -178.5589139
prob -195.5767289
prob -111.7414895
prob -225.8635395
prob -172.6071181
prob -151.5454483
prob -180.9792814
prob -205.9578489
prob -157.258973
prob -139.1955859
prob -118.1307194
prob -134.5142973
prob -121.6303608
prob -178.2524099
prob -118.4557329
prob -113.8645838
prob -168.241472
prob -124.0548622
prob -133.1287053
prob -134.5117343
prob -133.7416847
prob -151.5296141
prob -171.2553834
prob -133.8774008
prob -118.9944632
prob -186.2482486
prob -157.4307948
prob -171.0830967
prob -193.5440275
prob -165.997071
prob -111.2223398
prob -141.2496481
prob -110.0792185
prob -229.3512204
prob -155.8628868
prob -150.2582428
prob -124.1043847
prob -113.431133
prob -72.46020751
prob -117.2059479
prob -175.0235659
prob -124.1236991
prob -147.8135963
prob -211.9491355
prob -171.7896882
prob -144.8560508
prob -120.6786486
prob -78.15198526
prob -135.4364086
prob -155.1723217
prob -80.48345498
prob -108.624862
prob -198.3040768
prob -245.1903844
prob -186.2877639
prob -65.14778818
prob -112.0624202
prob -85.39094659
prob -98.54108632
prob -123.3668464
prob -211.344716
prob -108.8837011
prob -133.6075
prob -137.2151723
prob -131.3562865
prob -103.4766754
prob -136.7931094
prob -143.9819323
prob -210.8215432
prob -142.290061
prob -114.0157284
prob -125.3150189
prob -148.892631
prob -152.393896
prob -68.61289394
prob -166.8482012
prob -198.3136335
prob -127.4957047
prob -199.0758604
prob -147.4023508
prob -130.1063142
prob -158.4345761
prob -207.7082341
prob -137.893099
prob -173.2276306
prob -65.82391368
prob -137.0720312
prob -192.1851513
prob -183.5546256
prob -74.6802874
prob -212.277945
prob -173.4354532
prob -109.9857142
prob -103.3741391
prob -119.2770291
prob -163.9997598
prob -190.2059438
prob -108.4750947
prob -237.5395971
prob -153.5177958
prob -146.4485695
prob -176.0108266
prob -141.6648704
prob -129.4177466
prob -155.9320547
prob -193.7568497
prob -162.8891237
prob -119.0263041
prob -128.3481545
prob -190.7363692
prob -95.6966538
prob -239.8497703
prob -90.72271356
prob -132.7633888
prob -132.2060395
prob -144.2406325
prob -185.1122635
prob -134.1771151
prob -178.4308665
prob -94.31546531
prob -178.8680329
prob -124.0854191
prob -125.2616967
prob -107.4927557
prob -85.39122684
prob -118.4099517
prob -177.1691967
prob -179.3303491
prob -94.24397644
prob -171.6273623
prob -149.1116904
prob -189.9888323
prob -140.5316661
prob -238.6855449
prob -84.10312602
prob -191.5856761
prob -126.0147903
prob -191.458356
prob -124.8306922
prob -153.5930596
prob -155.0900308
prob -150.4219272
prob -249.6424834
prob -193.2026123
prob -110.1599103
prob -214.3342387
prob -133.104958
prob -156.0834555
prob -198.5171071
prob -111.9731496
prob -180.572232
prob -186.3880632
prob -395.3806062
prob -87.32469543
prob -88.44407406
prob -138.6234685
prob -178.5787038
prob -132.4599527
prob -211.7459667
prob -184.6366574
prob -152.3730065
prob -186.4279533
prob -94.39259276
prob -135.2089714
prob -124.0144461
prob -118.6074983
prob -159.2088521
prob -164.662206
prob -184.3831451
prob -171.2714001
prob -143.6904022
prob -84.88445774
prob -138.3912777
prob -186.7113933
prob -131.9290765
prob -208.8912866
prob -223.4877962
prob -103.003098
prob -86.07337504
prob -189.7142723
prob -128.0908632
prob -124.2139482
prob -117.4887947
prob -211.8854892
prob -138.255252
prob -190.2415127
prob -161.0844665
prob -168.1474265
prob -108.8742951
prob -213.5487119
prob -188.996791
prob -74.93474668
prob -143.0447759
prob -114.3977549
prob -131.6774658
prob -198.541545
prob -158.7229834
prob -144.5995285
prob -92.34122471
prob -121.0631457
prob -182.2386188
prob -117.8685105
prob -163.479293
prob -165.4572481
prob -204.9648512
prob -137.8644957
prob -107.0090986
prob -125.660803
prob -136.1128105
prob -245.9897533
prob -164.8072338
prob -95.71856112
prob -205.2680538
prob -178.4406165
prob -179.8565316
prob -106.6223941
prob -148.6187147
prob -101.6866484
prob -139.6772171
prob -148.8574987
prob -88.6835369
prob -171.0312567
prob -156.0417638
prob -126.2058596
prob -146.5297185
prob -124.9770123
prob -131.756918
prob -176.9382231
prob -108.7631371
prob -132.2272179
prob -99.11790438
prob -233.255996
prob -158.7826168
prob -179.1987255
prob -143.0777941
prob -105.1889445
prob -73.95547476
prob -227.7044138
prob -94.37728118
prob -101.6821162
prob -202.7480055
prob -129.1587784
prob -191.9438157
prob -71.22978717
prob -122.3901583
prob -197.7576247
prob -210.2195591
prob -162.9282523
prob -99.16468817
prob -96.18161264
prob -161.9159635
prob -200.2697706
prob -72.03866111
prob -82.80694777
prob -137.5975976
prob -255.7242972
prob -177.8971495
prob -203.6546337
prob -114.454839
prob -112.449773
prob -110.7084704
prob -138.9562141
prob -158.4087171
prob -218.3054577
prob -126.9195611
prob -206.7389926
prob -146.6825075
prob -192.174348
prob -168.7215536
prob -190.027179
prob -152.6581126
prob -160.568152
prob -194.1254878
prob -177.2640285
prob -207.4280357
prob -152.6210439
prob -154.230565
prob -143.4989139
prob -138.2436568
prob -138.436116
prob -135.0847166
prob -188.14686
prob -171.0060439
prob -91.40599013
prob -154.4728399
prob -95.0946251
prob -185.3319517
prob -154.2245388
prob -161.5837432
prob -176.773694
prob -122.9950832
prob -110.5564666
prob -85.58071338
prob -129.4959427
prob -179.0521055
prob -191.4540797
prob -231.1970524
prob -171.2517571
prob -199.0360492
prob -100.0299111
prob -201.2747663
prob -202.3606902
prob -133.9865342
prob -100.2719662
prob -170.0020648
prob -180.6440287
prob -110.0218437
prob -203.4799009
prob -120.5387646
prob -115.2264686
prob -73.10086912
prob -69.84044256
prob -112.8260435
prob -159.5808978
prob -215.4817575
prob -161.6486621
prob -114.9686285
prob -230.8069878
prob -138.0072156
numerical derivative 0 analytic 0
numerical derivative 0 analytic 0
numerical derivative 0 analytic 0
numerical derivative 0 analytic 0
numerical derivative 0 analytic 0
numerical derivative 0 analytic 0
numerical derivative 0 analytic 0
acceptance 0.6598157143
timesteps   avg   err
0.02028009948  -19471.91958  0.193363732
0.009088022176  -19471.33216  0.09004762545
0.03029496342  -19486.5948  0.1326068616
1.023791126  -755.5099321  53.54694747
0.1862459421  7.409149445  1.20192334
1.06162861  2022.859609  182.8923049
0.001825021446  3.115006291  0.7585935419

c[0] = -19471.91958 +/- 0.193363732
c[1] = -19471.33216 +/- 0.09004762545
c[2] = -19486.5948 +/- 0.1326068616
c[3] = -755.5099321 +/- 53.54694747
c[4] = 7.409149445 +/- 1.20192334
c[5] = 2022.859609 +/- 182.8923049
c[6] = 3.115006291 +/- 0.7585935419
Assessment of the fit:
fitted value -19469.48493 data -19469.06162 +/- 0.2386866021
fitted value -19472.58075 data -19472.91209 +/- 0.1824016432
fitted value -19473.40895 data -19473.40941 +/- 0.1821665035
fitted value -19473.33316 data -19473.66228 +/- 0.1540107974
fitted value -19473.1279 data -19473.23056 +/- 0.1770901115
fitted value -19472.56029 data -19471.3645 +/- 0.2461646212
fitted value -19473.83432 data -19473.74408 +/- 0.1834546702
fitted value -19473.25662 data -19473.50973 +/- 0.181134859
fitted value -19472.83754 data -19473.06261 +/- 0.1576947666
fitted value -19472.29316 data -19472.30654 +/- 0.1572564139
fitted value -19472.11391 data -19471.42344 +/- 0.1274873928
fitted value -19471.74118 data -19472.26098 +/- 0.1614681087
fitted value -19471.60483 data -19471.36149 +/- 0.1711038626
fitted value -19471.46683 data -19471.80548 +/- 0.1436593298
fitted value -19487.59765 data -19487.34243 +/- 0.2215584649
fitted value -19486.69404 data -19486.79659 +/- 0.1401297033
minimum skew nan
